pkg/server/replicas.go

```diff
@@ -81,7 +103,7 @@ func makeReplicaHandler(namespace string, kube kubernetes.Interface) http.Handle
 		}
 
 		opts := metav1.GetOptions{}
-		dep, err := kube.AppsV1().Deployments(namespace).Get(context.TODO(), functionName, opts)
+		dep, err := kube.AppsV1().Deployments(lookupNamespace).Get(context.TODO(), functionName, opts)
```

Comment:
we should probably pass the `r.Context()` here, but we should open a second PR for this kind of improvement